Zend/zend_gc.c

```diff
@@ -147,6 +149,12 @@ static void gc_trace_ref(zend_refcounted *ref) {
 }
 #endif
 
+typedef struct _gc_refcounted_stack {
+	int count;
+	int max;
```

Comment:
Using an unsigned integer type for these would be preferable. `size_t` would be semantically correct.

Reply:
Ok, will do that.